src/main/java/com/atlassian/db/replica/api/AuroraConnectionDetails.java

```diff
@@ -0,0 +1,44 @@
+package com.atlassian.db.replica.api;
+
+public final class AuroraConnectionDetails {
```

Comment:
There's nothing Aurora specific here. @tciesla-atlassian WDYT about renaming it to ConnectionDetails?